concourse/steps/rm_pr_label.mako

```diff
@@ -1,5 +1,5 @@
 <%def
-  name="rm_pr_label_step(job_step, job_variant, github_cfg, indent)",
+  name="rm_pr_label_step(job_step, job_variant, cfg_set, indent)",
```

Comment:
please rather pass github_cfg && concourse_cfg explicitly.

reasoning: this way, we do not implicitly depend on the default-cfg element from cfg-set, but accept a more explicit cfg, which makes the function more generic/flexible on the one-hand-side, and easier to understand (imo..) -> explicit is better than implicit